src/middlewares.js

```diff
@@ -325,7 +325,7 @@ export function handleParseErrors(err, req, res, next) {
 
     res.status(httpStatus);
     res.json({ code: err.code, error: err.message });
-    log.error('Parse error: ', err);
+    log.error(err.toString(), err);
```

Comment:
Shouldn't it be just `og.error(err.toString());`? Otherwise we are logging twice the same error message, for example, `Object not found.`.

Reply:
If we do so, we won't log the error stack. Wouldn't this be worse?

Reply:
I think the stack has not been logged as we can see [here](https://github.com/parse-community/parse-server/pull/6014/files#diff-ea8d3c7e6b7f64e4e1c8aead76e6ca45R274). In order to log the stack I think we should go with `log.error(err.toString(), err.stack);`. Can you please confirm?